### PR TITLE
Created the html to add a contact to hiring partners

### PR DESCRIPTION
**Pull request by Tibereon**

### CODE REVIEW

File: templates/hiringPartnerPage.phtml [hunk 1]
```diff
@@ -32,8 +32,34 @@
             <input id='companyUrl' name='companyUrl' class="submitHiringPartner form-control" type="text" placeholder="Company Website Url" data-max='255'>
             <input id="submitHiringPartner" type="submit" class="btn btn-success"/>
         </form>
-        <div id='messages'></div>
-        <h3>Current Hiring Partners</h3>
-        <div id='companies' ></div>
+    <div class="container">
+        <h1>Add contact to Hiring partner</h1>
+        <form id='addContactForm'>
```
Comment (mallowmew) [inline templates/hiringPartnerPage.phtml]: HTML/CSS IDs and class names should be kebab-case

File: src/Classes/ViewHelpers/SelectCompanyViewHelper.php [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+
+namespace Portal\ViewHelpers;
+
+
+class selectCompanyViewHelper
+{
+    public static function selectCompanyDropdown($data)
+    {
+        $companySizeDropdown = '';
```
Comment (mallowmew) [inline src/Classes/ViewHelpers/SelectCompanyViewHelper.php]: Should probably rename this variable for this context

File: templates/hiringPartnerPage.phtml [hunk 3]
```diff
@@ -32,8 +32,34 @@
...
+                <label class="form-check-label" for="defaultCheck1">
+                    Primary contact
+                </label>
+            </div>
+            <input id="submitHiringPartnerContact" type="submit" class="btn btn-success"/>
+        </form>
     </div>
+    <div id='messages'></div>
+    <h3>Current Hiring Partners</h3>
+    <div id='companies' ></div>
```
Comment (mallowmew) [inline templates/hiringPartnerPage.phtml]: Extra space after 'companies'

Comment (AlisonYoon) [reply templates/hiringPartnerPage.phtml]: There is still a space between 'companies' and the closing bracket >

**Pull request merged**